package.json

```diff
@@ -3,18 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "bootstrap": "^4.1.3",
+    "axios": "^0.20.0",
+    "bootstrap": "^4.5.2",
     "react": "^16.13.1",
     "react-cookie": "^3.0.4",
     "react-dom": "^16.13.1",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "1.1.5",
+    "react-scripts": "^3.4.3",
     "reactstrap": "^6.5.0"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "browserslist": {
```

Comment:
@jai1408 what is the need to add **browserlist** ?

#1 Please check this PR.

might wanna check below link.
[https://create-react-app.dev/docs/supported-browsers-features/](url)